documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -540,6 +540,10 @@
          "lock_uid":"00000000000000000000000000000000" // Only present when database is locked
       },
       "generation":2,
+      "gray_failure" : {
+         "excluded_processes" : [
```

Comment:
Should we rename this field to `excluded_servers`, that way we would use the same name as in the `configuration`  section.

Reply:
The reason I said processes was that the addresses here are ip+port, so you can have multiple such addresses being served from a single server. I was actually planning to change the terminology for gray failure related code from server to process because of this reason. But perhaps I am missing something - do you know why we use the term servers for ip+port addresses? 

Also did not realize that `configuration` section already has excluded servers, and there addresses are also ip+port. I think consistency of terminology is more important, so I agree that we should rename to server. Will do soon.

Reply:
Probably for historical reasons a process was referred to as a "server" but I'm not sure about the details here @jzhou77 might know more about the reason.

Reply:
Don't know the history here, and the "server" is the term we used.